assets/bingo/lang/en_us.json

```diff
@@ -395,7 +395,7 @@
 	"bingo.got_sniper_bingo.diag.top_left": "[%s] %s got a sniper bingo in the top left to bottom right diagonal!",
 	"bingo.got_sniper_bingo.diag.bottom_left": "[%s] %s got a sniper bingo in the bottom left to top right diagonal!",
 
-	"bingo.got_sniper_bingo.double.col_row": "[%s] %s got a sniper double bingo in row %s and col %s!",
+	"bingo.got_sniper_bingo.double.col_row": "[%s] %s got a sniper double bingo in col %s and row %s!",
```

Comment:
You missed the non-sniper version in this file

Reply:
Oh yeah, my bad 😅